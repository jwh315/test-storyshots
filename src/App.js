import React from 'react';
import {Virtuoso} from 'react-virtuoso';

function App() {
    const virtuoso = null;

    return (
        <div>
            <Virtuoso
                ref={virtuoso}
                style={{height: "50vh"}}
                item={index => <div>{index}</div>}
                totalCount={50}
            />
        </div>
    );
}

export default App;
