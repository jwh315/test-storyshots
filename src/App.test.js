import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import shallow from 'react-test-renderer/shallow';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('does snapshotting', () => {
  const renderer = shallow.createRenderer();
  const result = renderer.render(<App />);

  expect(result).toMatchSnapshot();
});
