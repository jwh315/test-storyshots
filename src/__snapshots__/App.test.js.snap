// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does snapshotting 1`] = `
<div>
  <Virtuoso
    item={[Function]}
    style={
      Object {
        "height": "50vh",
      }
    }
    totalCount={50}
  />
</div>
`;
